### ISSUE TITLE
Train classificator issue

### ISSUE THREAD

**Issue by 4ku** [rail-berkeley/serl#49]
```
Traceback (most recent call last):
  File "/ros_ws/src/serl/examples/pick_place_real/train_reward_classifier.py", line 160, in <module>
    app.run(main)
  File "/usr/local/lib/python3.10/dist-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.10/dist-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/ros_ws/src/serl/examples/pick_place_real/train_reward_classifier.py", line 97, in main
    classifier = create_classifier(key, sample["next_observations"], image_keys)
  File "/ros_ws/src/serl/serl_launcher/serl_launcher/networks/reward_classifier.py", line 74, in create_classifier
    new_params = classifier.params.unfreeze()
AttributeError: 'dict' object has no attribute 'unfreeze'
I0515 10:52:56.182127 134645912520512 core.py:560] signal_shutdown [atexit]
```

**Comment by 4ku** [comment]
Fixed with adding line `params = freeze(params) ` in `serl/serl_launcher/serl_launcher/networks/reward_classifier.pyreward_classifier.py`: 

```
    classifier_def = BinaryClassifier(encoder_def=encoder_def)
    params = classifier_def.init(key, sample)["params"]
    classifier_def = BinaryClassifier(encoder_def=encoder_def)
    params = freeze(params) 
    classifier = TrainState.create(
        apply_fn=classifier_def.apply,
        params=params,
        tx=optax.adam(learning_rate=1e-4),
    )
```

**Comment by 4ku** [comment]
Also there is another issue with saving the model:
```
Traceback (most recent call last):
  File "/ros_ws/src/serl/examples/pick_place_real/train_reward_classifier.py", line 160, in <module>
    app.run(main)
  File "/usr/local/lib/python3.10/dist-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.10/dist-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/ros_ws/src/serl/examples/pick_place_real/train_reward_classifier.py", line 151, in main
    checkpoints.save_checkpoint(
  File "/usr/local/lib/python3.10/dist-packages/flax/training/checkpoints.py", line 683, in save_checkpoint
    ocp.PyTreeCheckpointHandler(restore_with_serialized_types=False)
TypeError: PyTreeCheckpointHandler.__init__() got an unexpected keyword argument 'restore_with_serialized_types'
I0515 10:57:01.155173 126078530799424 core.py:560] signal_shutdown [atexit]
```

**Comment by 4ku** [comment]
Fixed with `pip install orbax-checkpoint==0.5.10`
